samples/Microsoft.Bot.Samples.Connector.EchoBot/Startup.cs

```diff
@@ -27,7 +32,14 @@ public Startup(IHostingEnvironment env)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();
+            services.AddBot<EchoBot>(options =>
```

Comment:
This is... beautiful. And very ASP.Net Core.

Reply:
interesting...

---
In reply to: [171711939](https://github.com/Microsoft/botbuilder-dotnet/pull/218#discussion_r171711939) [](ancestors = 171711939)